paper-api/src/main/java/org/bukkit/block/TrialSpawner.java

```diff
@@ -4,14 +4,41 @@
 import org.bukkit.entity.Entity;
 import org.bukkit.entity.Player;
 import org.bukkit.spawner.TrialSpawnerConfiguration;
-import org.jetbrains.annotations.ApiStatus;
 import org.jetbrains.annotations.NotNull;
 
 /**
  * Represents a captured state of a trial spawner.
  */
 public interface TrialSpawner extends TileState {
 
+    /**
+     * Gets the Gametime in ticks when the cooldown ends. 0 if not currently in cooldown.
+     *
+     * @return the Gametime in ticks
+     */
+    public long getCooldownEndsAt();
+
+    /**
+     * Sets the Gametime in ticks when the cooldown ends.
+     *
+     * @param ticks the GameTime in ticks for the new cooldown
+     */
+    public void setCooldownEndsAt(long ticks);
+
+    /**
+     * Gets the Gametime in ticks when the next spawn attempt happens. 0 if not currently active.
+     *
+     * @return the Gametime in ticks
+     */
+    public long getNextMobSpawnsAt();
```

Comment:
`getNextSpawnAttemptTick` or something?

Reply:
maybe just getNextSpawnAttempt

Reply:
Was a bit iffy on that because that could also be the next attempted mob or whatever.

Reply:
getNextSpawnAttemptTime maybe?